BuildAutomation/BuildOpenvSwitch.ps1

```diff
@@ -1,7 +1,7 @@
-Param(
-  [string]$OVSGitBranch = "branch-2.5-cloudbase",
-  [string]$SignX509Thumbprint
-)
+Param(
+  [string]$OVSGitBranch = "branch-2.5-cloudbase",
```

Comment:
^M replaced from my editor
\t as well
I brought it to the default it was before